src/bin/main.rs

```diff
@@ -1,5 +1,14 @@
 // Copyright 2019 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
+//! Parsec is the Platform AbstRaction for SECurity, a new open-source initiative to provide a
+//! common API to secure services in a platform-agnostic way.
+//!
+//! Parsec documentation is available at:
+//! https://parallaxsecond.github.io/parsec-book/index.html
+//!
+//! Most of Parsec configuration comes from its configuration file.
+//! Please check the documentation to find more about configuration:
+//! https://parallaxsecond.github.io/parsec-book/user_guides/configuration.html
```

Comment:
I think you can use these like markdown links:

```suggestion
//! Please check the documentation to find more about configuration
//! [here](https://parallaxsecond.github.io/parsec-book/user_guides/configuration.html)
```